snap/snapcraft.yaml

```diff
@@ -13,9 +13,15 @@ adopt-info: multipass
 confinement: strict
 base: core22
 
+architectures:
```

Comment:
What determines architecture support? Is it based on the backend drivers that we use?

Reply:
It's based on what Multipass can actually build on.  Without this, it would try to build on the default architectures on Launchpad (amd64, arm64, armhf, ppc64el, s390x), but it is broken on armhf due to the lack of `kvm` support there.  So now I made it explicit so it won't try to build on armhf and always fail.